### PR TITLE
Feature/notifications table

### PR DESCRIPTION
**Pull request by tnagorra**
Depends on https://github.com/idmc-labs/helix-server/pull/374

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: src/components/tableHelpers/ButtonAction.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React, { useCallback } from 'react';
```
Comment (tnagorra) [inline src/components/tableHelpers/ButtonAction.tsx:1]: Let's remove this component

File: src/components/tables/EventsEntriesFiguresTable/NudeEventsTable/AssigneeChangeForm/index.tsx [hunk 2]
```diff
@@ -76,8 +76,9 @@ const schema: FormSchema = {
 const defaultFormValues: PartialForm<FormType> = {};
 
 const roles: UserRole[] = [
-    'REGIONAL_COORDINATOR',
-    'MONITORING_EXPERT',
+    // FIXME:
+    // 'REGIONAL_COORDINATOR',
+    // 'MONITORING_EXPERT',
```
Comment (tnagorra) [inline src/components/tables/EventsEntriesFiguresTable/NudeEventsTable/AssigneeChangeForm/index.tsx:81]: Let's add why we are disabling this comment

File: src/views/Notifications/NotificationList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
+import React, { useMemo, useCallback } from 'react';
```
Comment (tnagorra) [inline src/views/Notifications/NotificationList/index.tsx:1]: Let's delete this component

Comment (tnagorra) [reply src/views/Notifications/NotificationList/index.tsx:1]: Done!

File: src/components/tableHelpers/ButtonAction.tsx (hunk 1 above)
Comment (tnagorra) [reply src/components/tableHelpers/ButtonAction.tsx:1]: Done!

**Review by priyesh777 — APPROVED** [review]

**Review by priyesh777 — APPROVED** [review]

File: src/components/Badge/index.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import { _cs, isNotDefined } from '@togglecorp/fujs';
+
+import styles from './styles.css';
+
+interface BadgeProps {
+    count: number | null | undefined;
+    className?: string;
+}
+
+function Badge(props: BadgeProps) {
```
Comment (AdityaKhatri) [inline src/components/Badge/index.tsx:11]: Can we rename this to notification badge?

**Review by AdityaKhatri — APPROVED** [review]

File: src/components/Badge/index.tsx (hunk 4 above)
Comment (tnagorra) [reply src/components/Badge/index.tsx:11]: This is a general badge. Not a notification badge.

**Pull request merged**